lib/challenge.js

```diff
@@ -275,10 +291,19 @@ let nestedArrays = [
 
 function prompt13() {
   // YOUR CODE HERE
+  for(let i = 0 ; i < nestedArrays.length ; i++){
+    for(let j = 0 ; j < nestedArrays.length ; j++){
+      for(let m = 0 ; m < nestedArrays.length ; m++){
+        console.log(`i: ${i} / j: ${j} / m: ${m}`)
+      }
+      return
+    }
+    return
+  }
 }
```

Comment:
What you wrote is a double nested loop and you only need a single nested loop since there is only on level of nested arrays in `nestedArrays`

sometimes I'll rewrite things so I can see what is going on for example:
`let nestedArrays = [ ["a", "b", "c"], ["d", "e", "f"], ["g", "h", "i"], ];`

This way it may be easier to see the arrays inside of the array

When I run your code I am getting an output of:
```
i: 0 / j: 0 / m: 0
i: 0 / j: 0 / m: 1
i: 0 / j: 0 / m: 2
```
The expected output is to print each letter individually to the console.  

Look at number 12 as an example and see if you can get this one working (no resubmit required)